code/index.html

```diff
@@ -2,21 +2,265 @@
+          </div>
+        </a>
+      </div>
+
+      <div class="small-news-container">
+        <a href="https://stackoverflow.com/c/technigo/questions">
+          <img src="https://images.unsplash.com/photo-1469571486292-0ba58a3f068b?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2100&q=80" alt="">
+          <div class="small-text">
+            <h3>Ask the Dev Community</h3>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+          </div>
+        </a>
+      </div>
+
+      <div class="small-news-container">
+        <a href="https://soshace.com/what-is-mentorship-in-web-development/">
+          <img src="https://images.unsplash.com/photo-1526958097901-5e6d742d3371?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=2100&q=80" alt="">
+          <div class="small-text">
+            <h3>Find A Mentor</h3>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+          </div>
+        </a>
+      </div>
+
+      <div class="small-news-container">
+        <a href="https://se.linkedin.com/jobs/frontend-developer-jobs?position=1&pageNum=0">
+          <img src="https://images.unsplash.com/photo-1455849318743-b2233052fcff?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2100&q=80" alt="">
+          <div class="small-text">
+            <h3>Find Work As A Front End Developer</h3>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+          </div>
+       </a>
+      </div>
+
+      <div class="small-news-container">
+        <a href="https://skillcrush.com/blog/skills-to-become-a-front-end-developer/">
+          <img src="https://images.unsplash.com/photo-1501159599894-155982264a55?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2100&q=80" alt="">
+          <div class="small-text">
+            <h3>Exactly What You Need To Know To Become A Front End Developer In 2021</h3>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+         </div>
```

Comment:
Maybe something like `small-news-text` or `small-news-content` would be more contextual (instead of `small-text`)